### ISSUE TITLE
How to select and install webdrivers

### ISSUE THREAD

**Issue by DougPuchalski** [titusfortner/webdrivers#55]
I'd expect the README to explain how to select which driver to use and how to get it installed.

**Comment by kapoorlakshya** [comment]
Hi @aceofspades, I am in the process of updating the README to get ready for the v3.7.2 release. Here is the draft - https://github.com/kapoorlakshya/webdrivers/blob/release_3_7_2/README.md

Let me know if you would like to see any other information in there.

**Comment by DougPuchalski** [comment]
@kapoorlakshya I think my particular problem was due to the latest version of the Chrome Browser not having a corresponding version of the Chromedriver. It wasn't clear what was going on until I dug in via the console and even then I only not HTTP errors until I figured out how to turn on verbose mode. So hopefully a future version can display better errors. To me it seems safe for any downloads to be logged without requiring verbose mode.
